_sass/layouts/guide.scss

```diff
@@ -1,26 +1,39 @@
-div.guide {
-  .toc {
-    float: right;
-    width: 350px;
-    background-color: #282a36;
-    border: 1px solid $cyan;
-    margin-left: 20px;
-    ul {
-      list-style: none;
+body:not(.guides-configuration-reference) {
+  div.guide {
+    padding-bottom: 6rem;
+
+    * {
+      color: #0D1C2C;
     }
     a {
-       text-decoration: none;
-       &:hover {
-         text-decoration: underline;
-       }
+      color: $link;
+    }
+    pre {
+      background-color: $white;
```

Comment:
We lost syntax color for code in the guides. e.g. http://localhost:4000/guides/getting-started-guide#the-jax-rs-resources

Reply:
Note also that the TIPS icon and maybe other icons don't look well on the gray background of guides. See right above http://localhost:4000/guides/getting-started-guide#using-injection

Reply:
As for syntax highlighting I'm not sure it's a new issue, it never worked locally for me.

Reply:
So it's a new issue, I can get syntax highlighting on the develop branch locally.